inc/zoo/swar/associative_iteration.h

```diff
@@ -419,10 +427,12 @@ constexpr auto multiplication_OverflowUnsafe_SpecificBitCount(
 
     auto halver = [](auto counts) {
         auto msbCleared = counts & ~S{S::MostSignificantBit};
-        return S{msbCleared.value() << 1};
+        T res = msbCleared.value() << 1;
+        return S{res};
```

Comment:
Why so many unnecessary changes?

Reply:
was getting seemingly random type errors.

Reply:
will try and resolve